Gemfile

```diff
@@ -46,5 +46,6 @@ gem 'asciidoctor', '>= 1.5.2'                           # If using AsciiDoc synt
 # gem 'kramdown', '>= 0.14.2'                             # If using Markdown
 # gem 'uglifier', '>= 1.3.0'                              # If using the minify transformer
 # gem 'htmlcompressor', '>= 0.0.3'                        # If using the minify transformer
+gem 'bootstrap-sass'
```

Comment:
Aren't you using bootstrap? In the next release, Awestruct won't depend on Bootstrap, or foundation, or 960, so you'll need it in your Gemfile.